webui/react/src/pages/WebhookList.module.scss

```diff
@@ -0,0 +1,20 @@
+.overflow {
+  padding: 6px;
+}
+.emptyBase {
+  margin-inline: auto;
+  margin-top: 150px;
+  max-width: 350px;
+  text-align: center;
+
+  .icon {
+    padding: 16px;
+  }
+  h4 {
+    margin-bottom: 8;
```

Comment:
you are missing `px`

Reply:
oh this is borrowed from ModelRegistry module scss

Reply:
i think it will fallback to `px` https://stackoverflow.com/a/11275156
prob i can fix it in https://github.com/determined-ai/determined/pull/5238

Reply:
great, 🙏  if you can update it here and in the ModelRegistry css